.travis.yml

```diff
@@ -19,9 +19,11 @@ env:
   - DB=mysql
 
 rvm:
+  - 2.5.1
```

Comment:
@aplegatt are you using this ruby version for specific reason ? or we could change that to `2.5` ?

Reply:
This was just a test.